source/control/request_receiver.hh

```diff
@@ -34,6 +37,7 @@ namespace psyllid
 
             void execute();
 
+            mv_accessible_noset( scarab::param_node, set_conditions );
```

Comment:
This should be `mv_referrable_const`.  That'll provide an access function `const scarab::param_node& set_condition() const` instead of the access-by-value set and get functions.